packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItems/FieldGuideItems.js

```diff
@@ -1,19 +1,21 @@
-import { Grid } from 'grommet'
+import { Box, Grid } from 'grommet'
 import React from 'react'
 import PropTypes from 'prop-types'
 import FieldGuideItemAnchor from './FieldGuideItemAnchor'
 
 export default function FieldGuideItems ({ className, items }) {
```

Comment:
The `export` statement should go at the bottom of the file

Reply:
Is this a style preference?

Reply:
Yup, it makes it easier to add HOCs later on